src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.test.js

```diff
@@ -2,15 +2,20 @@ import { screen, fireEvent } from '@testing-library/react';
 import { renderWithRouter } from 'src/utils/testHelpers';
 import mockSavedAccounts from '@tests/fixtures/accounts';
 import { truncateAddress } from '@wallet/utils/account';
+import { mockAuth } from 'src/modules/auth/__fixtures__';
+import { useAuth } from 'src/modules/auth/hooks/queries';
 import AccountManagementDropdown from './AccountManagementDropdown';
 
 const mockCurrentAccount = mockSavedAccounts[0];
 const mockOnMenuClick = jest.fn();
+jest.mock('@auth/hooks/queries');
 jest.mock('../../../account/hooks/useCurrentAccount.js', () => ({
   useCurrentAccount: jest.fn(() => [mockCurrentAccount]),
 }));
 
 describe('AccountManagementDropdown', () => {
+  useAuth.mockReturnValue({ data: mockAuth });
```

Comment:
why?
auth it should return the mock from msw why we mock it like this?

Reply:
reason is that it messes up and it's hard to control. We've tried it previously and we had to go with just mocking it. @soroushm